_co-op-companies/A-bold.md

```diff
@@ -7,16 +7,19 @@ image: "assets/img/logos/bold-icon.png"
 width: "100px"
 complete: true
 members:
-  - name: "Elizabeth Kenyon"
-    summary: "Elizabeth did her second work term at Bold."
-    statement: "She worked on Recurring Orders, working on a usability focused API."
-    image: "assets/img/co-op/elizabeth.jpg"
-  - name: "Julia Stoyko"
-    summary: "Julia did her first work term at Bold."
-    statement: "She worked on the Customer Pricing and Product Options applications, writing in PHP and React/Redux. She had an awesome time there."
-    image: "assets/img/co-op/julia.jpg"
-  - name: "Rebecca Tiessen"
-    summary: "Rebecca did her third work term at Bold."
-    statement: "She worked on the Brain app, which uses store data to make informed suggestions on increasing store revenue. She mainly worked in React/Redux and Go, and had an overall amazing time."
-    image: "/assets/img/co-op/rebecca.jpg"
+    - name: "Rebecca Tiessen"
+      summary: "Rebecca did her third work term at Bold."

```

Comment:
@shenyunw You changed everyone's summary to be the person's name, except for this one. Is that on purpose or should this summary be `"Rebecca Tiessen"`?